.github/workflows/build.yml

```diff
@@ -374,7 +374,7 @@ jobs:
       if: ${{ failure() }}
 
   publish:
-    needs: [tests, config-tests, test-cht-form, translations]
+    needs: [tests, config-tests, test-cht-form]
```

Comment:
Proposed this change in a [separate PR](https://github.com/medic/cht-core/pull/9984). Leaving this here for now to keep image builds while collecting missing translation strings.